docs/user/unattended/profile/software.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 2
+---
+
+# Additional software
+
+Depending on the [product you are installing](./product), Agama will select a set of patterns and
+packages to install. However, it is possible to add your own list of patterns and packages using the
+`software` section.
+
+This section supports two keys:
+
+- `patterns`: list of patterns to install (e.g., `gnome`, `kde`, `fips`, etc.). You can find a list
+  of patterns for your distribution using `zypper`.
```

Comment:
command is `zypper se --type pattern /.*/`

Reply:
OK, I have added the command. Although just `zypper se --type pattern` works for me.